doc/Tribler API.md

```diff
@@ -0,0 +1,31 @@
+# Tribler API
```

Comment:
Ok that's maybe better indeed. Do you agree if I rename the `api` package to `restapi`?